src/inverseRobot.js

```diff
@@ -24,7 +24,19 @@
  * @return {object}
  */
 function inverseRobot(robot) {
-  // write code here
+  const keys = Object.values(robot);
+  const vals = Object.keys(robot);
```

Comment:
Would be better to name variables more clear

```suggestion
  const robotKeys = Object.values(robot);
  const robotValues = Object.keys(robot);
```

Reply:
> Would be better to name variables more clear
Done!)